maat-orchestration/src/main/java/uk/gov/justice/laa/crime/orchestration/service/WorkflowPreProcessorService.java

```diff
@@ -47,9 +47,12 @@ private boolean hasApplicationStatusChanged(RepOrderDTO repOrderDTO, CaseDetailD
                 && !repOrderDTO.getRorsStatus().equalsIgnoreCase(status);
     }
 
-    public void preProcessEvidenceRequest(UserActionDTO userActionDTO) {
+    public void preProcessEvidenceRequest(UserActionDTO userActionDTO, boolean isUpliftChanged) {
         UserSummaryDTO userSummaryDTO = Optional.ofNullable(userActionDTO.getUsername())
                 .map(maatCourtDataService::getUserSummary).orElse(null);
         validationService.isUserActionValid(userActionDTO, userSummaryDTO);
+        if (isUpliftChanged) {
+            validationService.checkUpliftFieldPermissions(userSummaryDTO);
+        }
```

Comment:
Just wondering, is there a reason for the uplift check being seperate to the other user action checks within `isUserActionValid()`?

Reply:
I think its slightly different, there is an additional validation on the uplift fields. This part of the validation logic was already migrated, we are just invoking this as part of the current story.